cfme-versions.gemspec

```diff
@@ -1,29 +1,28 @@
-pwd = File.expand_path("../", __FILE__)
-$LOAD_PATH.unshift(pwd) unless $LOAD_PATH.include?(pwd)
-require "cfme-versions"
+
+lib = File.expand_path("../lib", __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require "cfme/versions/version"
 
 Gem::Specification.new do |spec|
   spec.name          = "cfme-versions"
-  spec.version       = CFME::Versions::version
+  spec.version       = CFME::Versions::GEM_VERSION
```

Comment:
This was by design.  There is no reason to add a 4th version number to keep track of here, and split up the files for this project even further.

If you really want a distinct version for this project, you should then instead change this in the `cfme-versions.rb` file as well.

Reply:
Ah...I can switch that...didn't realize it was by design